.github/workflows/MarkdownLinksCheck.yml

```diff
@@ -3,19 +3,6 @@ name: Check Markdown links
 on: 
   pull_request:
 
-permissions:
-  contents: read
-  
 jobs:
   markdown-link-check:
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v2
-      with:
-        ref: ${{ github.event.pull_request.head.sha }}
-    - uses: gaurav-nelson/github-action-markdown-link-check@v1
-      with:
-        use-quiet-mode: 'yes'
-        use-verbose-mode: 'yes'
-        check-modified-files-only: ${{ github.event_name == 'pull_request' }}
-        config-file: '.github/workflows/mlc_config.json'
+    uses: Open-Systems-Pharmacology/Workflows/.github/workflows/MarkdownLinksCheck.yml@main
```

Comment:
That's all I need to do: refer to the global workflow (here using the global workflow in the **main** branch of the repo).
If I change the configuration of the global workflow - it will be automatically propagated to this repo, so no change required here

Reply:
Why are there so many files changed in this PR?

Reply:
Just wrongly created PR into master instead into V11 first :)